src/Application/View/Extension/DataHelpersExtension.php

```diff
@@ -267,29 +266,6 @@ public function getEnhancedHtml($string, array $params = [])
         return RichText::getEnhancedHtml($string, $params);
     }
 
-    /**
-     * Return verbatim value for an itemtype field.
-     * Returned value will be unsanitized if it has been transformed by GLPI sanitizing process (value fetched from DB).
-     * Twig autoescaping system will then ensure that value is correctly escaped in rendered HTML.
-     *
-     * @param mixed  $string
-     *
-     * @return mixed
-     *
-     * @deprecated 11.0
```

Comment:
This method is deprecated since v11; it should not be removed before next major (same for several other cases in current PR)